### PR TITLE
ci: Update references from master branch to main

### PR DESCRIPTION
**Pull request by jarojasm95**
Follow up from: https://github.com/flakeheaven/flakeheaven/issues/25

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 templates_path = ['_templates']
 source_suffix = ['.rst', '.md']
-master_doc = 'index'
+main_doc = 'index'
```
Comment (pwoolvett) [inline docs/conf.py:27]:
```suggestion
root_doc = 'index'
```

maybe ?
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-master_doc

File: docs/conf.py [hunk 2]
```diff
@@ -73,7 +73,7 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-    (master_doc, 'flakeheaven.tex', 'FlakeHeaven Documentation',
+    (main_doc, 'flakeheaven.tex', 'FlakeHeaven Documentation',
```
Comment (pwoolvett) [inline docs/conf.py:76]:
```suggestion
    (root_doc, 'flakeheaven.tex', 'FlakeHeaven Documentation',
```

same, https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-master_doc

File: docs/conf.py [hunk 3]
```diff
@@ -82,7 +82,7 @@
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [(master_doc, 'flakeheaven', 'FlakeHeaven Documentation', [author], 1)]
+man_pages = [(main_doc, 'flakeheaven', 'FlakeHeaven Documentation', [author], 1)]
```
Comment (pwoolvett) [inline docs/conf.py:85]:
```suggestion
man_pages = [(root_doc, 'flakeheaven', 'FlakeHeaven Documentation', [author], 1)]
```

same, https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-master_doc

File: docs/conf.py [hunk 4]
```diff
@@ -91,7 +91,7 @@
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, 'flakeheaven', 'FlakeHeaven Documentation',
+    (main_doc, 'flakeheaven', 'FlakeHeaven Documentation',
```
Comment (pwoolvett) [inline docs/conf.py:94]:
```suggestion
    (root_doc, 'flakeheaven', 'FlakeHeaven Documentation',
```

last one, https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-master_doc

**Comment by jarojasm95** [comment]
once #43 is merged and this branch is updated, these tests _should_ be pass

**Review by pwoolvett — DISMISSED** [review]

**Review by mcarans — APPROVED** [review]
Looks good to me

**Pull request merged**